bin/utils/setup-log-dir.sh

```diff
@@ -44,7 +44,7 @@ get_install_log_directory() {
 
   INSTALL_LOG_DIR=$(get_full_path "${INSTALL_LOG_DIR}")
 
-  if { [[ ! -d "${INSTALL_LOG_DIR}" ]] || [[ ! -r "${INSTALL_LOG_DIR}" ]] }
+  if [[ ! -d "${INSTALL_LOG_DIR}" ]] || [[ ! -r "${INSTALL_LOG_DIR}" ]]
```

Comment:
Have to be careful about this, as this may be a bash vs sh incompatibility. I havent tried it, but if it proves to be an issue then it may need to be specified differently.